### PR TITLE
[STRATCONN-3100] TikTok Docs Update

### PR DESCRIPTION
**Pull request by marinhero**
### Proposed changes

We've updated the TikTok destinations. This PR reflects the new state of the destination.

### Merge timing

- Depending on the rollout of the destination. Targeting September 28th.

### CODE REVIEW

**Review by rvadera12 — COMMENTED** [review]
Great Job! I left a few comments and @maryamsharif could you just fact check me for a couple of comments.

File: src/connections/destinations/catalog/actions-tiktok-audiences/index.md [hunk 1]
```diff
@@ -73,4 +79,32 @@ The setup is complete and the audience will start syncing to TikTok. The audienc
...
+6. Once authenticated, toggle “Enable Destination” on and click  **Save Changes**.
+
+7. Navigate to the **Mappings** tab, click **New Mapping**, and select **Add to Audience**.
+
+8. Repeat Steps 6 and 7 to also set up a **Remove from Audience** mapping.
+
+9. Navigate back to **Engage > Audiences** and click on the audience from Step 1.
+
+10. Click **Add Destinations** and select the TikTok Audiences destination you just created. 
+    In the settings that appear in the side panel, toggle the **Send Track** option on and do **not** change the Audience Entered/Audience Exited event names.
```
[the remark this answers is not in the document]
Comment (rvadera12) [reply src/connections/destinations/catalog/actions-tiktok-audiences/index.md:103]: Confirmed with Maryam that technically you can change the event name - so lets add 

_If you update the event names, please make sure to also update it in the Add Users and Remove Users mappings under step 1 **(Select events to map and Send)**._

File: src/connections/destinations/catalog/actions-tiktok-audiences/index.md [hunk 2]
```diff
@@ -12,38 +12,44 @@ By using Segment's TikTok Audiences destination, you can increase traffic and dr
 
 ## Getting started
 
+### Note
+
+- If you created a TikTok Audiences destination instance before September 18th your instance and all subsequent new instances are considered _legacy_. Therefore, follow the documentation labeled as such. Newer instances are considered _native_.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-tiktok-audiences/index.md:17]:
```suggestion
- If you created a TikTok Audiences destination instance before September 18th, 2023, your instance(s) and all subsequent instances are considered _legacy_ instances. To create new _legacy_ instances, see the [Create a TikTok audience (Legacy)](#create-a-tiktok-audience-legacy) documentation. Users who created their first instance after September 18, 2023 are considered to have _native_ instances. To create new _native_ instances, see the [Configure the TikTok Audiences destination (Native)](#configure-the-tiktok-audiences-destination-native) documentation.
```

File: src/connections/destinations/catalog/actions-tiktok-audiences/index.md [hunk 3]
```diff
@@ -12,38 +12,44 @@ By using Segment's TikTok Audiences destination, you can increase traffic and dr
...
+- Both versions offer the same set of features, the difference sits in the way to configure them. For legacy, you are required to create an audience manually whereas native will do that automatically. Additionally, manual configuration is required per action when using legacy, native will perform this configuration as well.
+- For more information about how to update from _legacy_ to _native_ reach out to [friends@segment.com](mailto:friends@segment.com).
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-tiktok-audiences/index.md:19]:
```suggestion
- For more information about how to update from _legacy_ to _native_, reach out to [friends@segment.com](mailto:friends@segment.com).
```

File: src/connections/destinations/catalog/actions-tiktok-audiences/index.md [hunk 4]
```diff
@@ -12,38 +12,44 @@ By using Segment's TikTok Audiences destination, you can increase traffic and dr
 
 ## Getting started
 
+### Note
+
+- If you created a TikTok Audiences destination instance before September 18th your instance and all subsequent new instances are considered _legacy_. Therefore, follow the documentation labeled as such. Newer instances are considered _native_.
+- Both versions offer the same set of features, the difference sits in the way to configure them. For legacy, you are required to create an audience manually whereas native will do that automatically. Additionally, manual configuration is required per action when using legacy, native will perform this configuration as well.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-tiktok-audiences/index.md:18]:
```suggestion
- Both _legacy_ and _native_ instances have the same set of features, but are configured differently. Legacy instances require you to create an audience or action manually, but native instances automatically create audiences and actions.
```

File: src/connections/destinations/catalog/actions-tiktok-audiences/index.md [hunk 5]
```diff
@@ -12,38 +12,44 @@ By using Segment's TikTok Audiences destination, you can increase traffic and dr
...
+you do not need to update the status of the mapping to `enabled`.
 
-You do not need to update the status of the mapping to `enabled`.
+see tiktok's [create/delete an audience segment](https://ads.tiktok.com/marketing_api/docs?id=1739940583739393){:target="_blank"} for instructions on how to create a tiktok audience segment. 
 
-See TikTok's [Create/Delete an audience segment](https://ads.tiktok.com/marketing_api/docs?id=1739940583739393){:target="_blank"} for instructions on how to create a TikTok audience segment. 
+### Connect the TikTok Audiences destination (_legacy_)
 
-### Connect the TikTok Audiences destination
+**add user and remove user are considered legacy actions**
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-tiktok-audiences/index.md:52]: Why are they considered legacy actions? Does that mean they're restricted to the legacy instances of this destination?

```suggestion
> info ""
> Add User and Remove User are considered legacy actions.
```

File: src/connections/destinations/catalog/actions-tiktok-audiences/index.md [hunk 6]
```diff
@@ -73,4 +79,32 @@ The setup is complete and the audience will start syncing to TikTok. The audienc
 
 To sync additional audiences from your Engage space, create a separate mapping in the TikTok Audiences destination. Navigate to **Connections > Destinations**, search and select the TikTok Audiences destination, and follow Steps 7-11 above.
 
+### Configure the TikTok Audiences destination (_native_)
+
+1. From the Segment web app, navigate to **Engage > Audiences**. Ensure you are in the Engage space you plan to use with the TikTok Audiences destination. Either choose an existing Engage audience or create a new one. This is the audience you plan to send to TikTok.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-tiktok-audiences/index.md:84]:
```suggestion
1. From the Segment web app, navigate to **Engage > Audiences**. Ensure you are in the Engage space you plan to use with the TikTok Audiences destination. Either choose an existing Engage audience or create a new one.
```

File: src/connections/destinations/catalog/actions-tiktok-audiences/index.md [hunk 7]
```diff
@@ -73,4 +79,32 @@ The setup is complete and the audience will start syncing to TikTok. The audienc
...
+
+2. Navigate to **Engage > Engage Settings** and click **Destinations**. Please ensure you are still in the correct Engage space.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-tiktok-audiences/index.md:86]:
```suggestion
2. Navigate to **Engage > Engage Settings** and click **Destinations**. 
```

File: src/connections/destinations/catalog/actions-tiktok-audiences/index.md [hunk 8]
```diff
@@ -73,4 +79,32 @@ The setup is complete and the audience will start syncing to TikTok. The audienc
...
+
+3. Search for “TikTok Audiences” and select the destination. Click **Configure TikTok Audiences**.
+
+4. On the Select Source screen, your Engage space should already be selected as the source. Click **Confirm Source**.
+
+5. On the Destination **Settings** tab, name your destination and authenticate with TikTok Audiences using OAuth.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-tiktok-audiences/index.md:92]:
```suggestion
5. On the **Settings** tab for the TikTok Audiences destination, name your destination and authenticate with TikTok Audiences using OAuth.
```

File: src/connections/destinations/catalog/actions-tiktok-audiences/index.md [hunk 9]
```diff
@@ -73,4 +79,32 @@ The setup is complete and the audience will start syncing to TikTok. The audienc
...
+
+6. Once authenticated, toggle “Enable Destination” on and click  **Save Changes**.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-tiktok-audiences/index.md:94]:
```suggestion
6. Once authenticated, toggle “Enable Destination” on and click **Save Changes**.
```

File: src/connections/destinations/catalog/actions-tiktok-audiences/index.md [hunk 10]
```diff
@@ -73,4 +79,32 @@ The setup is complete and the audience will start syncing to TikTok. The audienc
...
+
+7. Navigate to the **Mappings** tab, click **New Mapping**, and select **Add to Audience**.
+
+8. Repeat Steps 6 and 7 to also set up a **Remove from Audience** mapping.
+
+9. Navigate back to **Engage > Audiences** and click on the audience from Step 1.
+
+10. Click **Add Destinations** and select the TikTok Audiences destination you just created. 
+    In the settings that appear in the side panel, toggle the **Send Track** option on and do **not** change the Audience Entered/Audience Exited event names. 
+    Provide the [**Advertiser ID**](https://ads.tiktok.com/help/article/ad-account-information-faq?lang=en) linked to the TikTok account that will receive the audience data, as well as the **ID Type** of data you'll be sending. Click **Save Settings**.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-tiktok-audiences/index.md:104]:
```suggestion
    Provide the [**Advertiser ID**](https://ads.tiktok.com/help/article/ad-account-information-faq?lang=en) linked to the TikTok account that will receive the audience data, as well as the **ID Type** of data you'll be sending. Click **Save Settings**.
```
```suggestion
    Provide the [**Advertiser ID**](https://ads.tiktok.com/help/article/ad-account-information-faq?lang=en){:target="_blank”} linked to the TikTok account that will receive the audience data, as well as the **ID Type** of data you'll be sending. Click **Save Settings**.
```

**Review by forstisabella — CHANGES_REQUESTED** [review]

**Comment by forstisabella** [comment]
@marinhero - just want to confirm new deploy date is looking like 9/25?

File: src/connections/destinations/catalog/actions-tiktok-audiences/index.md [hunk 11]
```diff
@@ -12,38 +12,44 @@ By using Segment's TikTok Audiences destination, you can increase traffic and dr
...
-3. Fill in the mappings on the Select mappings panel accordingly.
-4. On the Send test event panel, click **Test Mapping**.
-5. You've created your audience. Take note of the `audience_id` from the response as you will need it when you create additional mappings. 
+1. navigate to new mapping and select **create audience**. 
+2. on the add test event panel, click **load sample event**.
+3. fill in the mappings on the select mappings panel accordingly.
+4. on the send test event panel, click **test mapping**.
+5. you've created your audience. take note of the `audience_id` from the response as you will need it when you create additional mappings. 
 
+you can use the same mapping to create as many audiences as you would like. you just need to change the audience name and click **test mapping** again.
```
[the remark this answers is not in the document]
Comment (rvadera12) [reply src/connections/destinations/catalog/actions-tiktok-audiences/index.md:42]: This is going to be a field on a mapping and we already listed above the user can use the same mapping, so there technically isn't a screen name besides mapping and the customer shouldn't have any confusion.

File: src/connections/destinations/catalog/actions-tiktok-audiences/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rvadera12) [reply src/connections/destinations/catalog/actions-tiktok-audiences/index.md:103]:
```suggestion
    In the settings that appear in the side panel, toggle the **Send Track** option on and do **not** change the Audience Entered/Audience Exited event names. 
    
    *If you update the events names, please make sure to also update it in the "Add to Audience" and "Remove from Audience" mappings under step 1 (Select events to map and Send).*

    
```

**Comment by marinhero** [comment]
Hi @forstisabella! Yes, 9/25 is the accurate date. Thank you and @rvadera12 for addressing the suggestions 🙇🏽

File: src/connections/destinations/catalog/actions-tiktok-audiences/index.md [hunk 12]
```diff
@@ -73,4 +80,35 @@ The setup is complete and the audience will start syncing to TikTok. The audienc
...
+8. Repeat Steps 6 and 7 to also set up a **Remove from Audience** mapping.
+
+9. Navigate back to **Engage > Audiences** and click on the audience from Step 1.
+
+10. Click **Add Destinations** and select the TikTok Audiences destination you just created. 
+    In the settings that appear in the side panel, toggle the **Send Track** option on and **Send Identify** option off. 
+    
+    *If you update the events names from the default Audience Entered/Audience Exited, please make sure to also update it in the "Add to Audience" and "Remove from Audience" mappings under step 1 (Select events to map and Send).*
+
+    Provide the [**Advertiser ID**](https://ads.tiktok.com/help/article/ad-account-information-faq?lang=en){:target="_blank”} linked to the TikTok account that will receive the audience data, as well as the **ID Type** of data you'll be sending. Click **Save Settings**.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/actions-tiktok-audiences/index.md:108]:
```suggestion
    In the settings that appear in the side panel, toggle the **Send Track** option on and **Send Identify** option off. Provide the [**Advertiser ID**](https://ads.tiktok.com/help/article/ad-account-information-faq?lang=en){:target="_blank”} linked to the TikTok account that will receive the audience data, as well as the **ID Type** of data you'll be sending. Click **Save Settings**.
    
> info " "
> If you update the events names from the default Audience Entered/Audience Exited, please make sure to also update it in the "Add to Audience" and "Remove from Audience" mappings under step 1 (Select events to map and Send)
```

**Review by forstisabella — COMMENTED** [review]
A few changes and we'll be good to go for tomorrow 🚀

**Review by forstisabella — APPROVED** [review]

**Pull request merged**